pkgs/tools/admin/pulumi/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "pulumi-${version}";
+  version = "v0.16.2";
+
+  src = if (stdenv.system == "x86_64-linux")
+    then fetchurl {
+      url = "https://get.pulumi.com/releases/sdk/pulumi-${version}-linux-x64.tar.gz";
+      sha256 = "16qgy2pj3xkf1adi3882fpsl99jwsm19111fi5vzh1xqf39sg549";
+    }
+    else if (stdenv.system == "x86_64-darwin") then fetchurl {
+      url = "https://get.pulumi.com/releases/sdk/pulumi-${version}-darwin-x64.tar.gz";
+      sha256 = "18ck9khspa0x798bdlwk8dzylbsq7s35xmla8yasd9qqlab1yy1a";
+    }
+    else throw "platform not ${stdenv.system} supported!";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp * $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://pulumi.io/;
+    description = "Pulumi is a cloud development platform that makes creating cloud programs easy and productive.";
```

Comment:
No period at the end of the short description
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Reply:
there is